data/console/samba_ca_cert.pem

```diff
@@ -1,34 +1,24 @@
-UGQgNouu+2SPZwINvnARSLKVzxnlQfNSBEC3pIDTcH330t28t/jTpkV4PjASwkfV
-wzayVJ/5bjCNdHiev1znK/FoIw7vcexwL/o8FjmEpAExnR2tV124o1Cx1FjGqU37
-A4OlpSaPmkBH5DME1RANTiCp98VD/WDXVNj1sf4TqYtypdug2xeM+stSrDuSXFho
-K/isBYBZutrz/2oY/Z3BwA9SBW0vbNr4BToZJBig60+TyopRJqaTOFAjx+J7Vfy3
-aVAGKHcGK6aNwNxpSS7WrRSM6iJvokrOWL1I/wnHrWPo5p1U5WJatLWfBuLjQdTv
-9jvi1ONLF2kL1w176o61l2YHvh6qrl4laguifhtXT89RgIrvAToM6XnZYT9YvXiM
-nXZ3qUyq92BKykIhXim6khOi2xyco87tx09pfNRVqcP+qkd/5Efq6a85naEyRREw
-FBDr/q/xjasF4uD/bHUXsuYWRbLWnLVCY07AUzLGCTm01RdlOn+1GHsGdEIjdjTN
-OeVuxClRlp6pGsnQ4GBdjFOsMickY3OiUkNhK7tx0nBb3Al8maGpdERDLKV8PfQi
-9gTvbiHn67bS0Bz/Ub34aMztRjJXemOxf3b9HFaAdxLlQdgDxEBdLzmEs4W/yPFa
-DFhqtbuNAgMBAAGjYzBhMB0GA1UdDgQWBBTVclm7/kNGcKPGi8lkKMDXLm16CjAf
-BgNVHSMEGDAWgBTVclm7/kNGcKPGi8lkKMDXLm16CjAPBgNVHRMBAf8EBTADAQH/
-MA4GA1UdDwEB/wQEAwIBhjANBgkqhkiG9w0BAQsFAAOCAgEAxNQoybF99+2WgJlY
-nfWmTJMIC4oIdvG3zVXPB5WP/MAGO89x5wCQHo3+qVVDfJiqlnQRbReen+FOJ7VD
-6CxsxH2G2uJwT0g6OxJZ3+m2oA+aMFC363lr8mMgPpUmKkB6zTm/XyTEqoUKsWND
-uKT0GCxTwwNSCB93OPOTYnHMjUdtRfNSQ/3gdML/wcl12LnV18wqz/5ineHYbfwW
-nJOL+jMZODHWeCQPZrZzv4uYx6hpnWEISemo0OcnJ9iboGCnMN3fISvn1OyHmhzY
-X/TfBCy7rineqBES3q0JPUy1WjOKE+C14qPg6u5zO7yg2WCEXe5fdRvetU5lDDar
-bt6Xi5h4NdasnV1+dSnO86GUP23VW1mlHSDpm9uaYelSZJdaTAectUueGzs6xq8+
-9sQjpUqajeCXFA9E9PP93DPryxgydN/wTqStvULfBwgpXF7yg8JsaW1ZaaGfMkzJ
-R99curwlbVq0dq+rlkZzrak0BUrL2h0q7sgc4IgEgufr+weAHvx5+u674XCU6IvH
-bv7sImi7ok6jXzM273dzkIJMbcukP52+OMqUdp3+NoZCR877KSl9fTkCyE0rrLoE
-mvqpZDdlrXw+OrpFA3nd9wGboMN3lL4v7CiImDjmjZNVjuEC1HXA1WoyWZ9NJqPQ
-u3bGZ/G5IvkqJVy2DcQdLNzi37I=
+MIID/DCCAuSgAwIBAgIJANpRVf/rqXkFMA0GCSqGSIb3DQEBCwUAMIGKMQswCQYD
+VQQGEwJERTEPMA0GA1UECAwGQmF5ZXJuMRIwEAYDVQQHDAlOdXJlbWJlcmcxDTAL
+BgNVBAoMBFN1c2UxCzAJBgNVBAsMAlFBMRwwGgYDVQQDDBNvcGVucWEubGRhcHRl
+c3Qub3JnMRwwGgYJKoZIhvcNAQkBFg1jYV9xYUBzdXNlLmRlMB4XDTIwMDUyMjEz
+NDE1MVoXDTQwMDUxNzEzNDE1MVowgYoxCzAJBgNVBAYTAkRFMQ8wDQYDVQQIDAZC
+YXllcm4xEjAQBgNVBAcMCU51cmVtYmVyZzENMAsGA1UECgwEU3VzZTELMAkGA1UE
+CwwCUUExHDAaBgNVBAMME29wZW5xYS5sZGFwdGVzdC5vcmcxHDAaBgkqhkiG9w0B
+CQEWDWNhX3FhQHN1c2UuZGUwggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIB
+AQCjehFt5RbBveE9AY507kTTxLvh95WmqrYUlCDIAgtI6xmD/l5tSsyFAzlkw+R1
+6NDLsZUmWhO0sgSnzAxb2ocwB+H4wBmnOJwAy0Ij/4O7OCxcWLsIiyMwLWfrmNld
+8OpHnnYvuolt4enX7uJp6fGF+d87frf51SaIX0iJdIpKf2Flaq6OVZjRMBP4BOXS
+rXBIxcQIpaJVu0m+C8nw617SO65wqd308A8noC8w+RyIzjxy1DkSJrAVGLYKSGeS
+1H4jNLA1TFeb1pJqeD3n2LOk1Q3IJ2PJpI+8sp2IzJP7AuRCKxR+umTgU8WblRuA
+BNa6EQwtni2cSmzkeRFoXNkNAgMBAAGjYzBhMB0GA1UdDgQWBBTgfdZoKrqMXXAE
+Lj6QbWEr9ICgADAfBgNVHSMEGDAWgBTgfdZoKrqMXXAELj6QbWEr9ICgADAPBgNV
+HRMBAf8EBTADAQH/MA4GA1UdDwEB/wQEAwIBhjANBgkqhkiG9w0BAQsFAAOCAQEA
+clwRww1ezHYb0m3H7pMBa0mOM41YolRrCFTdQrxcI1dodG4ugNSBJzJJy/e8HuAj
+u96f//NdG4cei1/ASnz6Iw6StoVv4NK+hfgTBWDzerSd/dKt22AgFxNdnEusmGPx
+M50NHzWz7AqTkjRrerqEMOjRm9LeXor4nH+umAn+LG4h6gfVfoBvV6YiCKgLpxIF
+U1AfbnN3xwj3nt3wGMjTUQqko6JVOEf/dT1lDV9RFl7KyhhRedntBDcMELJEhLoz
+Oxg3tA+gBgjDZVi5Qb/CKEUiwci67S5LipfewrKV6ytK6sxhetp8ublCEt/fqLdi
+dFutIWQ/1JHnJIz3I8qBmw==
 -----END CERTIFICATE-----
```

Comment:
Why do you do this?

Reply:
There was some problem for github.com from my side. 
let me check with it.

Reply:
@pdostal  fix this problem, please check it again